master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -265,6 +265,10 @@ private[celeborn] class Master(
       }).sum()
   }
 
+  masterSource.addGauge(MasterSource.SHUFFLE_FALLBACK_COUNT) { () =>
```

Comment:
Although the gauge can describe the total count of fallback, I think the meters metrics will be more helpful in showing the status of current clusters and jobs.
The speed of fallback can be more intuitive.
https://metrics.dropwizard.io/4.2.0/manual/core.html#meters

Reply:
thanks for the comments, checking